src/app/pages/MediaArticlePage/SecondaryColumn.tsx

```diff
@@ -2,12 +2,20 @@
 /** @jsx jsx */
 import path from 'ramda/src/path';
 import { jsx } from '@emotion/react';
-import { articleDataPropTypes } from '#models/propTypes/article';
+import { articleDataPropTypes } from '#app/models/propTypes/article';
```

Comment:
Since we're updating the path, can we try to move away from the # notation for imports, and use a relative import instead?